e2e-test/trace/build.gradle

```diff
@@ -233,7 +233,7 @@ dependencies {
     integrationTestImplementation project(':data-prepper-plugins:opensearch')
     integrationTestImplementation project(':data-prepper-plugins:otel-trace-group-processor')
     integrationTestImplementation 'org.awaitility:awaitility:4.2.0'
-    integrationTestImplementation "io.opentelemetry.proto:opentelemetry-proto:${versionMap.opentelemetryProto}"
+    integrationTestImplementation "io.opentelemetry.proto:opentelemetry-proto:${targetOpenTelemetryVersion}"
```

Comment:
Where is `targetOpenTelemetryVersion` coming from? I know that `versionMap.opentelemetryProto` is defined [here](https://github.com/opensearch-project/data-prepper/blob/5f492a5cf41b1248429d110ffa4b6e273399e9d7/build-resources.gradle#L10)

Reply:
It's coming from e2e-test/build.gradle. See the change to this file.